packages/sdks/snippets/react/src/components/CustomTabs.tsx

```diff
@@ -71,4 +73,59 @@ const CustomTabs = (props: TabProps) => {
   );
 };
 
-export default CustomTabs;
+export const customTabsInfo: RegisteredComponent = {
+  component: CustomTabs,
+  name: 'TabFields',
+  /** To receieve Builder props inside your custom component: by default false  */
+  shouldReceiveBuilderProps: {
+    /** To access builder's Blocks relative to your parent */
```

Comment:
this isn't gramatically correct. Also, I find it weird that the comments start with `to`:

```suggestion
  shouldReceiveBuilderProps: {
    /** enabling this causes the SDK to pass the `builderBlock` prop down to the component */
```